src/utils/breachResolution.js

```diff
@@ -109,19 +110,24 @@ const breachResolutionDataTypes = {
  */
 function appendBreachResolutionChecklist (userBreachData, options = {}) {
   const { verifiedEmails } = userBreachData
+
   for (const { breaches } of verifiedEmails) {
     breaches.forEach(b => {
       const dataClasses = b.DataClasses
+      const blockList = (AppConstants.HIBP_BREACH_DOMAIN_BLOCKLIST ?? '').split(',')
+      const showLink = b.Domain && !blockList.includes(b.Domain)
+
       const args = {
         companyName: b.Name,
+        breachedCompanyLink: showLink ? `https://${b.Domain}` : '',
```

Comment:
aside: in my personal testing, I think `http://` had better results than `https://` (somewhere between 5-10% more 2xx/3xx results).

Reply:
Interesting, thanks for the note. With us trying to be cautious where we link out to I think I’d feel more comfortable linking out to `https://`.